pkg/cli/client/server_info_cmd.go

```diff
@@ -0,0 +1,150 @@
+//go:build search
+// +build search
+
+package client
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+
+	zerr "zotregistry.io/zot/errors"
+	"zotregistry.io/zot/pkg/api/constants"
+)
+
+func NewServerInfoCommand() *cobra.Command {
+	serverInfoCmd := &cobra.Command{
+		Use:   "status",
+		Short: "Information about the server configuration and build information",
+		Long:  `Information about the server configuration and build information`,
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			searchConfig, err := GetSearchConfigFromFlags(cmd, NewSearchService())
+			if err != nil {
+				return err
+			}
+
+			return GetServerInfo(searchConfig)
+		},
+	}
+
+	serverInfoCmd.PersistentFlags().String(URLFlag, "",
+		"Specify zot server URL if config-name is not mentioned")
+	serverInfoCmd.PersistentFlags().StringP(ConfigFlag, "c", "",
+		"Specify the registry configuration to use for connection")
+	serverInfoCmd.PersistentFlags().StringP(UserFlag, "u", "",
+		`User Credentials of zot server in "username:password" format`)
+	serverInfoCmd.Flags().StringP(OutputFormatFlag, "f", "text", "Specify the output format [text|json|yaml]")
+
+	return serverInfoCmd
+}
+
+const (
+	StatusOnline  = "online"
+	StatusOffline = "offline"
+)
+
+func GetServerInfo(config SearchConfig) error {
+	username, password := getUsernameAndPassword(config.User)
+	ctx := context.Background()
+
+	serverInfo := ServerInfo{}
+
+	mgmtEndpoint, err := combineServerAndEndpointURL(config.ServURL, fmt.Sprintf("%s%s",
+		constants.RoutePrefix, constants.ExtMgmt))
+	if err != nil {
+		return err
+	}
+
+	_, err = makeGETRequest(ctx, mgmtEndpoint, username, password, config.VerifyTLS, config.Debug,
+		&serverInfo, config.ResultWriter)
+	switch true {
+	case err == nil:
+		serverInfo.Status = StatusOnline
+	case errors.Is(err, zerr.ErrPageNotFound):
+		serverInfo.Status = StatusOnline
```

Comment:
I'm not convinced about this. How do we know this is the correct url of a zot server? Maybe there's another app running there.

Reply:
How can we know for sure it's a zot server?

Reply:
I think we should https://github.com/opencontainers/distribution-spec/blob/main/spec.md#determining-support

Reply:
This is a liveness check, and not a "are you truly a zot or not" check

Reply:
I was thinking the user may have misconfigured the server url.